core/src/main/java/io/aiven/klaw/helpers/HandleDbRequests.java

```diff
@@ -2,12 +2,7 @@
 
 import io.aiven.klaw.dao.*;
 import io.aiven.klaw.error.KlawNotAuthorizedException;
-import io.aiven.klaw.model.enums.AclType;
-import io.aiven.klaw.model.enums.KafkaClustersType;
-import io.aiven.klaw.model.enums.OperationalRequestType;
-import io.aiven.klaw.model.enums.RequestMode;
-import io.aiven.klaw.model.enums.RequestOperationType;
-import io.aiven.klaw.model.enums.RequestStatus;
+import io.aiven.klaw.model.enums.*;
```

Comment:
If you want to double check this wild card import

Reply:
Thanks my intellij settings for this got reset 👎 have put them back to normal now and will push up these updates in a sec